### PR TITLE
Add copy/paste feature

### PR DESCRIPTION
**Pull request by LaContra**
https://trello.com/c/HEMUm7QC

copy/paste feature

### CODE REVIEW

File: common/actions.js [hunk 1]
```diff
@@ -443,6 +443,28 @@ export const saveKeys = () => {
...
+const paste = (elements) => {
+  return {
+    type: "PASTE",
+    elements
+  }
+}
+
+export const parseAndPaste = () => {
+  return (dispatch) => {
+    const nodesInfo = removeEdgesAndAssignNewIds(SessionStorage.getCopiedNodes(), intentId)
```
Comment (jingwanhsu) [inline common/actions.js:462]: What's the reason to remove edge?

File: cy_canvas/reducers/cyElements.js [hunk 2]
```diff
@@ -267,6 +267,11 @@ const cyElements = (state = [], action) => {
       )
     case "UNDO":
       return SessionStorage.popState()
+    case "COPY":
+      SessionStorage.saveCopiedNodes(action.elements)
+      return state
+    case "PASTE":
+      return state.concat(action.elements)
```
Comment (jingwanhsu) [inline cy_canvas/reducers/cyElements.js:274]: It could be better to update the selected status 💃

File: common/actions.js (hunk 1 above)
Comment (LaContra) [inline common/actions.js:462]: We talked about only support node copy/paste in the slack channel.

File: cy_canvas/reducers/cyElements.js (hunk 2 above)
Comment (LaContra) [inline cy_canvas/reducers/cyElements.js:274]: I'll leave this to another card to handle all the situation that involves selected status change.

**Pull request merged**